### DISCUSSION TITLE
Release process

### DISCUSSION THREAD

**Discussion by fermentfan** [golevelup/nestjs#710] [General]
Hey, I was just wondering about the state of CI/CD here in this repository especially when it comes to releasing. Is it currently a manual process triggered on a local machine? Are you looking for a setup to automate this? This comes up as I want to update the Stripe golevelup package, but it has not been released since a month or so.

**Comment by underfisk** [comment]
Hi @fermentfan the release process is still manually performed by @WonderPanda 
We were looking back ago to get Nx and possibly automate releases

**Comment by WonderPanda** [comment]
I'm definitely open to having some kind of automated release process or looking at expanding access to the NPM organization so that underfisk can publish releases without me.

**Reply by aeza8i** [reply]
Hi @WonderPanda,
Auto publishing is definitely the best idea. I hope this happens soon.
Is the next manual release imminent?
I can't wait for this to happen.
Thanks
